README.md

```diff
@@ -1 +1,19 @@
-# Fill me in with information
+Official MongoDB Node.js Driver Quick Start
+===========================================
+
+You can use this starter project to follow along the
+[MongoDB Node.js Driver Quick Start](https://docs.mongodb.com/drivers/node/current/quick-start/).
```

Comment:
Comment: I believe we generally do not capitalize "[driver](https://github.com/mongodb/docs-node/blob/master/source/index.txt#L27-L28)". I believe this also applies in the `package.json` description field if accepted.

```suggestion
[MongoDB Node.js driver Quick Start](https://docs.mongodb.com/drivers/node/current/quick-start/).
```

Reply:
This is part of a title which is why it is capitalized.